app.js

```diff
@@ -22,10 +26,12 @@ import translation from './modules/translation/translation';
 
 // Initialize Olm as early as possible.
 if (window.Olm) {
-    window.Olm.init().catch(e => {
-        console.error('Failed to initialize Olm, E2EE will be disabled', e);
-        delete window.Olm;
-    });
+  window.Olm.init().catch(e => {
+    console.error('Failed to initialize Olm, E2EE will be disabled', e);
+    delete window.Olm;
+    // Use toggleE2EE(false) to disable E2EE
+    store.dispatch(toggleE2EE(false)); 
```

Comment:
`store` might not exist at this point, that's partially why we are doing this so early. Did you run into a situation where this failed _after_ you were able to turn e2ee on?